spec/features/custom_crawler_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+# require 'custom_crawler'
+
+xdescribe "CustomCrawler" do
+  describe ".recursive_get" do
+    let (:url)        { "http://www.example.com" }
+    let (:cw)         { Cobweb.new }
+    let (:body)       { cw.get(url)[:body] }
+    let (:ccrawler)   { CustomCrawler.new }
+    let (:page_count) { Page.all.count }
+
+    it "increases the number of page objects by 1" do
+      ccrawler.get(url, 0)
```

Comment:
make sure you are not actually making web requests here.  Take a look at the [FakeWeb gem](https://github.com/chrisk/fakeweb) to mock out the http requests.  You may also want to use Fake to generate some random urls.